Source/WebKit/NetworkProcess/webtransport/cocoa/NetworkTransportSessionCocoa.mm

```diff
@@ -46,11 +47,12 @@
 }
 
 #if HAVE(WEB_TRANSPORT)
-static RetainPtr<nw_parameters_t> createParameters()
+static RetainPtr<nw_parameters_t> createParameters(WebCore::SecurityOriginData&& origin)
 {
-    auto configureWebTransport = [](nw_protocol_options_t options) {
+    auto configureWebTransport = [&origin](nw_protocol_options_t options) {
```

Comment:
This needs to be captured by value and not by reference because the stack that contains origin is likely gone by the time it is used.